src/app/offer-form/offer-form.component.html

```diff
@@ -0,0 +1,31 @@
+<form (ngSubmit)="onSubmit.emit(data)">
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12">
+        <label><b i18n>Tytuł ogłoszenia </b> <span i18n>(max 80 znaków): </span></label>
+        <mat-form-field appearance="outline">
+          <input matInput maxlength="80" placeholder="Wpisz tytuł ogłosznia..." i18n-placeholder required
+                 name="title"
+                 [(ngModel)]="data.title"/>
```

Comment:
Use reactive forms.

Reply:
I'm not sure about that for these simple forms we're doing here. [Angular docs ](https://angular.io/guide/forms-overview#choosing-an-approach) say:

> If you have very basic form requirements and logic that can be managed solely in the template, template-driven forms could be a good fit.

If you have a really firm opinion here, then sure - I'll switch.